### PR TITLE
List of submissions + Answers

### PR DESCRIPTION
**Pull request by yohanology**
*User can see list of forms in submissions
*User can see list of submissions for each form
*User can see answers for each submission

However for the list of answers, I made a http request inside a loop to get the label of the content for each answer... xD

### CODE REVIEW

File: www/js/controllers.js [hunk 1]
```diff
@@ -105,6 +116,69 @@ angular.module('starter.controllers', [])
...
+      $scope.answers = resp.answers;
+
+      // Get content label for each answer
+      $scope.answers.forEach(function(answer){
+
+        var contentId = answer.content_id;
+        
+        $http.get(apiUrl + "/contents/" + contentId).success(function(resp){
+          console.log(resp);
+          answer.label = resp.label
```
Comment (dshcheung) [inline www/js/controllers.js:172]: can you make sure all answer.label is correctly defined? In some cases when using `$http` inside a loop and defining `answer.label` might end up making all `answer.label` equal the 1st response. Meaning all your `answer.label` is the same. If not then you can ignore this message

File: www/js/controllers.js [hunk 2]
```diff
@@ -69,27 +73,34 @@ angular.module('starter.controllers', [])
...
+      console.log("answers", answers)
+
+      var hash = {answers: answers}
+      
+      // Submit the hash to submissions
+      $http.post(apiUrl + '/submissions/' + submissionId + '/answers', hash).success(function(response) {
+        console.log("response", response);
+      });
 
+      // Popup success after submitting
```
Comment (f3r) [inline www/js/controllers.js:103]: perhaps you should include the popUp show inside the `post` callback?

**Comment by f3r** [comment]
@yohanology can you ping me up when you review @dshcheung and my comments?

File: www/js/controllers.js (hunk 2 above)
Comment (yohanology) [inline www/js/controllers.js:103]: I fixed this in another branch that i will pr just after this.

File: www/js/controllers.js (hunk 1 above)
Comment (yohanology) [inline www/js/controllers.js:172]: So far we tested in local, and it's working... so I think we can ignore for the moment.

**Comment by yohanology** [comment]
done

**Pull request merged**